Cargo.toml

```diff
@@ -111,6 +111,7 @@ serde_json = "1.0"
 rand_xorshift = "0.3"
 rand_isaac = "0.3"
 criterion = { version = "0.4", features = ["html_reports"] }
+nalgebra = { path = ".", features = ["debug", "compare", "rand", "macros"]}
```

Comment:
That’s an interesting trick!

Reply:
Thanks. Did it because rust-analyzer was giving errors whenever I opened the `tests` folder, since it only parses the code reachable with the default features